packages/material-ui/src/Fab/Fab.d.ts

```diff
@@ -37,7 +37,7 @@ export type FabTypeMap<P = {}, D extends React.ElementType = 'button'> = ExtendB
     /**
      * The variant to use.
      */
-    variant?: 'round' | 'extended';
+    variant?: 'circular' | 'extended';
```

Comment:
We need to keep both `round` and `circular` when deprecating, otherwise, we are breaking the ts build.